src/Symfony/Component/Validator/ConstraintValidatorFactory.php

```diff
@@ -42,7 +39,7 @@ public function getInstance(Constraint $constraint)
 
         if (!isset($this->validators[$className])) {
             $this->validators[$className] = 'validator.expression' === $className
```

Comment:
Do we need to keep this special name `validator.expression`? Or can I just remove that here and change the `validatedBy` implementation of `ExpressionValidator`. I don't see why there is this edge case for this.